klt-cert-manager/config/rbac/role.yaml

```diff
@@ -57,3 +33,23 @@ rules:
   - get
   - list
   - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
+  name: cert-manager-role
+  namespace: keptn-lifecycle-toolkit-system
```

Comment:
this is added automagically by kubebuilder thanks to the annotation in line 37 of the controller, I added this role so that the our certificate manager will have only access to secrets in our namespace